src/assets/css/_sass/elements/_buttons.scss

```diff
@@ -5,3 +5,23 @@
   Don't forget hover, active states
 
 */
+
+.button {
+  padding: .575rem .75rem;
+  text-decoration: none;
+  box-shadow: 2px 2px 1px #1a1a1a;
+}
+
+.button:hover {
+  background-color: $color-primary--yellow-hover;
+  -webkit-transition: all .5s ease;
+  -moz-transition: all .5s ease;
+  -o-transition: all .5s ease;
+  -ms-transition: all .5s ease;
+  transition: all .5s ease;
```

Comment:
Use the transition mixin
`@include transition`

https://github.com/Canary-PrintDesign/canswe/blob/9507d0b1c55b618bcd1c022ab08d2ce7f892ed97/src/assets/css/_sass/tools/_transitions.scss#L13-L18
https://github.com/Canary-PrintDesign/canswe/blob/9507d0b1c55b618bcd1c022ab08d2ce7f892ed97/src/assets/css/_sass/settings/_core.scss#L14